res/controllers/MVave-SMC-Mixer-scripts.js

```diff
@@ -130,40 +130,42 @@ var SMCMixer;
         }
     }
 
-    class EqRack {
+    class EqRack extends components.ComponentContainer {
         constructor(index) {
+            super({});
             const channel = mapIndexToChannel(index);
             this.knob = new Encoder({
                 group: `[Channel${channel}]`,
                 midi: [0xB0, 0x10 + index],
                 inKey: "pregain",
             });
 
+            const _this = this;
             const btnInToggle = () => {
-                const knob = this.knob;
-                const origGroup = this.knob.group;
-                const origInKey = this.knob.inKey;
+                const knob = _this.knob;
+                const origGroup = _this.knob.group;
+                const origInKey = _this.knob.inKey;
                 return function() {
-                    if (this.isLongPressed) {
-                        if (knob.inKey === this.inKey.replace("button_", "") || (this.inKey === "enabled" && this.knob.inKey === "super1")) {
+                    if (_this.isLongPressed) {
+                        if (knob.inKey === _this.inKey.replace("button_", "") || (_this.inKey === "enabled" && _this.knob.inKey === "super1")) {
```

Comment:
wait, shouldn't it have been `knob` instead of `this` in the first place?

Reply:
oh yah, I guess it could have been. I forgot I redeclared it at the top (though also I have no idea why I did that, writing "this.knob" a few times isn't that much longer or harder). Fixed anyways. Thanks!